ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -310,9 +312,19 @@ + (ADAuthenticationResult *)processBrokerResponse:(NSURL *)response
         MSIDOauth2Factory *factory = [MSIDAADV1Oauth2Factory new];
         MSIDDefaultTokenCacheAccessor *otherAccessor = [[MSIDDefaultTokenCacheAccessor alloc] initWithDataSource:dataSource otherCacheAccessors:nil factory:factory];
         MSIDLegacyTokenCacheAccessor *cache = [[MSIDLegacyTokenCacheAccessor alloc] initWithDataSource:dataSource otherCacheAccessors:@[otherAccessor] factory:factory];
+        
+        NSString *userId = [[[result tokenCacheItem] userInformation] userId];
+        NSString *applicationidentifier = [ADRequestParameters applicationIdentifierWithAuthority:brokerResponse.authority];
+        NSString *enrollmentId = nil;
+        
+        if (applicationidentifier)
+        {
+            enrollmentId = [ADEnrollmentGateway enrollmentIDForHomeAccountId:nil userID:userId error:nil];
```

Comment:
do we want to provide home account here as well?

Reply:
addressed :)